dagshub/streaming/filesystem.py

```diff
@@ -245,7 +246,9 @@ def _special_file(self):
         # TODO Include more information in this file
         return b'v0\n'
 
-    def open(self, file: Union[PathLike, int], mode: str = 'r', *args, opener=None, **kwargs):
+    def open(self, file: Union[bytes, PathLike, int], mode: str = 'r', *args, opener=None, **kwargs):
+        if type(file) is bytes:
+            file = file.decode("utf-8")
```

Comment:
Why?

Reply:
yolov5 tried to open a file to write with a binary path. Apparently this is an accepted behavior as per docs